sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificateImportOptions.java

```diff
@@ -94,7 +94,7 @@ public CertificatePolicy certificatePolicy() {
      * @param certificatePolicy the management policy for the certificate
      * @return the CertificateImportOptions itself
```

Comment:
@return the updated CertificateImportOptions object.
here and below

Reply:
fixed.